client/src/components/cv/ViewCV.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
+import { Layout, Space } from 'antd';
+import { LoadingScreen } from 'components/LoadingScreen';
+import { MainSection, AboutSection, CoursesSection, FeedbackSection } from 'components/cv/sections';
+import { Contacts, UserData, CourseData, GetCVData } from '../../../../common/models/cv';
+import { PublicFeedback } from '../../../../common/models/profile';
+import { UserService } from 'services/user';
+
+const { Content } = Layout;
+
+type Props = {
+  ownerId: string;
+};
+
+type State = {
+  isLoading: boolean;
+  contactsList: Contacts | null;
+  userData: UserData | null;
+  coursesData: CourseData[] | null;
+  publicFeedback: PublicFeedback[] | null;
+  expires: number | null;
+};
+
+class ViewCV extends React.Component<Props, State> {
```

Comment:
please convert this component to functional component with hooks.
`componentDidMount` could be replaced with 
```typescript
const fetchData = useCallback(async () =>  {
...
}, [...]);

useEffect(() => {
  fetchData();
}, [])
```